console-subscriber/src/lib.rs

```diff
@@ -304,3 +287,37 @@ impl proto::tasks::tasks_server::Tasks for Server {
         Ok(tonic::Response::new(stream))
     }
 }
+
+impl Visit for FieldsCollector {
```

Comment:
nit, take it or leave it: I might call this `FieldVisitor` or something --- in `tracing` 0.2, the name "Collector" will refer to what are currently called "Subscribers", so this could be a little confusing in the future.